spec/core/ics-026-routing-module/UPGRADES.md

```diff
@@ -0,0 +1,141 @@
+# Application Upgrade Callbacks
+
+## Synopsis
+
+This standard document specifies the interfaces and state machine logic that IBC applications must implement in order to enable existing channels to upgrade their applications after the initial channel handshake.
```

Comment:
Same confusion as with the title. 
> enable existing channels to upgrade their applications

Is this true? Or are the callbacks necessary to enable the endpoint modules to adapt to a channel upgrade.?

Reply:
> enable existing channels to upgrade their applications

this is true. It will allow a transfer v1 to upgrade to breaking version transfer v2 using the channel upgrade process.